tautulli.json

```
@@ -0,0 +1,48 @@
+{
+    "Tautulli": {
+        "containers": {
+            "tautulli": {
+                "image": "tautulli/tautulli",
+                "launch_order": 1,
+                "ports": {
+                    "8181": {
+                        "description": "WebUI port. Suggested default: 8181",
+                        "host_default": 8181,
+                        "label": "WebUI port",
+                        "ui": true
+                    }
+                },
+                "volumes": {
+                    "/config": {
+                        "description": "Choose a Share for Tautulli configuration. E.g.: create a Share called tautulli-config for this purpose alone.",
+                        "label": "Config Storage"
+                    },
+                    "/plex-config": {
+                        "description": "Choose a Share that holds your Plex log files. This is typically the share that was selected to hold the plex configuration.This will allow Tautulli to read the Plex log files.",
```

Minor typo and style suggestions, mostly aimed at reinforcing the fact that Plex should probably be installed and setup before installing Tautulli.
```suggestion
                        "description": "Select the Share holding your Plex log files. This is typically the share that was selected to hold the Plex configuration. This will allow Tautulli to read the Plex log files.",
```